contracts/Colony.sol

```diff
@@ -61,6 +61,23 @@ contract Colony is ColonyStorage {
     });
   }
 
+  function bootstrapColony(uint _amountPerUser, address[] _users) public
+  auth
+  isInBootstrapPhase
+  {
+    colonyBootstrapped = true;
+
+    uint diff = (_amountPerUser * _users.length) - token.balanceOf(this);
+    if (diff > 0) {
+      token.mint(diff);
+    }
+
+    for (uint i = 0; i < _users.length; i++) {
+      token.transfer(_users[i], _amountPerUser);
+      IColonyNetwork(colonyNetworkAddress).appendReputationUpdateLog(_users[i], int(_amountPerUser), 1);
```

Comment:
The skill that this reputation is being added to should be the skill related to the colony's top-level domain, which will be stored in `domains[1].skillId` as you said.

This cast to `int` is required, because the reputationUpdateLog requires an `int` for that parameter. However, there is no guard to check that it is not going to underflow. We should insert one, or require the input to be of type `int` and then require that it is not below zero (and then cast it to `uint` for the token transfer).

Reply:
Yea, beginner mistake with the `int`s.